modules/ROOT/pages/introduction/intro-connector-configuration-overview.adoc

```diff
@@ -0,0 +1,148 @@
+
+[[test-connectivity]]
+=== Test the Connectivity of the Connector
+
+Both Studio and Flow Designer enable you to test the connection between a connector and its data source by clicking a button.
+
+In Studio, after you configure the required connection values in the *Global Element Properties* screen, click *Test Connection*:
+
+.Click the *Test Connection* button in Studio to test the connection
+image::introduction/intro-config-overview-studio-test.png[Test the connection in Studio]
+
+In Flow Designer, after you configure the required connection values in the *Configure Connection* screen, click *Test*:
+
+.Click the *Test* button in Flow Designer to test the connection
+image::introduction/intro-config-overview-fd-test.png[Test the connection in Flow Designer]
+
+[[reconnection-strategy]]
+=== Configure a Reconnection Strategy for the Connector
+
+When a Mule app starts, a connectivity test runs on connectors that must connect to an external server. If the test fails, the default behavior is to log a warning message and continue with the deployment of the app, instead of blocking the deployment.
+
+You can modify this behavior by setting the number of reconnection attempts to try after an initial failure and forcing the deployment to fail when all connection attempts fail.
+
+To configure a reconnection strategy for a connector using Studio, use the *Advanced* tab in the global element configuration *General* tab. The following example configures a reconnection strategy that:
+
+* Forces the deployment to fail if the connection fails and any configured reconnection attempts are exhausted
+* Limits the number of reconnection attempts to 5 at a frequency of every 4000 milliseconds (ms)
+
+.Use the fields in the Reconnection section of the global element to specify a reconnection strategy
+image::introduction/intro-config-overview-reconnection-strategy-studio.png[Reconnection strategy example using Studio]
+
+To configure a reconnection strategy for a connector using Flow Designer, use the *Advanced* tab in the global element configuration. The following example configures a reconnection strategy that:
+
+* Forces the deployment to fail if the connection fails and any configured reconnection attempts are exhausted.
+* Limits the number of reconnection attempts to 3 at a frequency of every 5000 milliseconds (ms).
+
+.Use the fields in the Reconnection Strategy section in Flow Designer to specify reconnection options
+image::introduction/intro-config-overview-reconnection-strategy-fd.png[Reconnection strategy example using Flow Designer]
+
+To determine whether a specific connector supports a reconnection strategy and to obtain information about reconnection strategy settings, see the reference guide for the connector.
+
+== Methods for Configuring Connectors
+
+You can configure a connector using Studio, Flow Designer, or XML.
```

Comment:
I wonder if we couldn't replace this with a statement in the introduction to the topic set.